static/js/tools/scatter/context.ts

```diff
@@ -176,6 +179,7 @@ function useContextStore(): ContextType {
   const [y, setY] = useState(EmptyAxis);
   const [place, setPlace] = useState(EmptyPlace);
   const [isQuadrants, setQuadrants] = useState(false);
```

Comment:
nit: maybe change to showQuadrants and showLabels to be consistent?

Reply:
done!